lib/nero/swap/swap_estimate.rb

```diff
@@ -0,0 +1,68 @@
+require 'date'
+
+module Nero
+  module Swap
+    class SwapEstimate < SimpleDelegator
+      INITIAL_PRIOR = 2e-4
+      REJECTION_THRESHOLD = 1e-07
+      DETECTION_THRESHOLD = 0.95
+
+      def adjust(agent, guess)
+        pl = agent.pl
+        pd = agent.pd
+
+        if guess=="LENS"
+          likelihood = pl
+          likelihood /= (pl*probability + (1-pd)*(1-probability))
```

Comment:
Surrounding space missing for operator '*'.<br>Surrounding space missing for operator '-'.